ci/valgrind-check/Cargo.toml

```diff
@@ -23,6 +23,7 @@ description = "Internal crate for zenoh."
 
 [dependencies]
 base64ct = "~1.6.0"
+zerofrom = "~0.1.5"
```

Comment:
Using `~0.1.5` doesn't pin the version. See https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#tilde-requirements
To pin to an exact version you must use `=0.1.5`.
The same applies to `base64ct` dependency.

Reply:
Thanks, my bad! Fixed in 6b5b0cd